content/en/community/hackathons/sessions/contributing-to-unikraft/content/Practical-Work_07.-Additional-Requirements.md

```diff
@@ -1 +1,3 @@
 Check the original library's `README` to see if it needs to be configured first, and add the proper rule if so.
+
+**Hint**: As in most ported libraries, a configuration step is required and it consists in simply running a script.
```

Comment:
I'd also hint at checking the `Config.uk` file for this task.

Reply:
How would the `Config.uk` file help with configuration steps?
Is it about the requirement for `newlib`?
If so, I would add it [here](https://unikraft.org/community/hackathons/usoc22/contributing-to-unikraft/#08-intermediary-check), in the intermediary check section, since the file that will need changes is the app's `Makefile`.

Reply:
There is a TODO_7 placed in `Config.uk` regarding adding `newlib` as a dependecy; it took me a while to discover that, because it wasn't specifically indicated to take a look into that file for the 7th task, I totally forgot about it:
```
# TODO_1
menuconfig LIBNAME
	bool "kdtree - simplify the use of kd-trees"
	default n
	# TODO_7
```

Reply:
Oh I see, you're right, I will add that as a note.